src/GraphQL.Conventions/Types/Resolution/ObjectReflector.cs

```diff
@@ -356,7 +356,17 @@ private static bool HasValidReturnType(MemberInfo memberInfo)
             {
                 return true;
             }
-            return returnType != typeof(object);
+                   
+            return 
+                returnType != typeof(object) && 
+                ObjectReflectorSettings.IgnoreFieldsWithVoidReturnType ? returnType != typeof(void) : true;
```

Comment:
Can you please rebase and pull in my change from https://github.com/graphql-dotnet/conventions/pull/78?